scripts/py_matter_idl/matter_idl/generators/kotlin/MatterClusters.jinja

```diff
@@ -210,7 +210,7 @@ class {{cluster.name}}Cluster(private val controller: MatterController, private
 {% if command.is_timed_invoke -%}
     val timeoutMs: Duration = Duration.ofMillis(timedInvokeTimeoutMs.toLong())
 {%- else -%}
-    val timeoutMs: Duration = timedInvokeTimeoutMs?.let { Duration.ofMillis(it.toLong()) } ?: Duration.ZERO 
+    val timeoutMs: Duration? = timedInvokeTimeoutMs?.let { Duration.ofMillis(it.toLong()) }
```

Comment:
Suggest some changes to make the code more consistent:
  - timeout is already a Duration, so 'ms' suffix is not required.
  - since we cast in a duration, we may as well accept input already in a duration format

```jinja
...
  ,  timedInvokeTimeout: Duration
  {%- if command.is_timed_invoke -%}? = null {% - endif -%}
)
....
```